pkg/ccl/backupccl/testdata/backup-restore/schedule-privileges

```diff
@@ -95,8 +95,12 @@ subtest end
 
 subtest control-schedules-privileges
 
+exec-sql
+CREATE SCHEDULE foocluster_admin FOR BACKUP INTO 'external://foo/cluster2' RECURRING '@hourly';
```

Comment:
nit: create this schedule later in the test, where it's used? And give it a new label like foocluster_admin_revoke?